src/recipe/recipe.service.ts

```diff
@@ -45,4 +49,14 @@ export class RecipeService {
       throw new NotFoundException();
     }
   }
+
+  async deleteRecipe(id: number) {
+    try {
+      await this.prisma.recipe.delete({
```

Comment:
it is ok but you should think about softDelete because we prefer to mark data in DB as deleted than really delete something

Reply:
to implement softDelete you should add an optional parameter to your `Recipe` model `deletedAt` and type `DateTime`

and then you have to take care to not returned softDeleted recipes in get recipe endpoint

Reply:
but implementing `softDelete` is optional now and you can skip it